service/src/main/resources/db/changesets/20220721_change_agent_email_and_subject_id_columns.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+- changeSet:
+    id: add change agent email and subject id column
+    author: yuhuyoyo
+    changes:
+    - addColumn:
+        tableName: workspace_activity_log
+        columns:
+        - column:
+            name: change_agent_email
```

Comment:
i was thinking it may get confused as if this is the change_agent or the change_target. But i guess we are preferring to have a json blob for the change target so it wouldn't get confused.

Reply:
Good point, how about actor_email?

- I know what "actor" means, I don't know what "agent" means
- Could we take out "change"?
- Code should be consistent. Eg ActivityLogChangeDetails should match what's here